pkg/frontend/querymiddleware/roundtrip.go

```diff
@@ -313,6 +313,9 @@ func newQueryMiddlewares(
 	queryStatsMiddleware := newQueryStatsMiddleware(registerer, engine)
 
 	remoteReadMiddleware = append(remoteReadMiddleware,
+		// Track query range statistics. Added first before any subsequent middleware modifies the request.
```

Comment:
I would rephrase this comment given the sanitizer runs before the queryStatsMiddleware.

Reply:
removed sanitizer